sdk/provisioning/Azure.Provisioning.AppConfiguration/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
```

Comment:
I thought we wanted these to be inside the same service dir as the mgmt package so that release is tied together.  Whenever you release mgmt you release provisioning too?  I was expecting some ci.yml changes to reflect this.

Reply:
See https://github.com/Azure/azure-sdk-for-net/pull/43045#discussion_r1544861966

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/43163